options/predict_options.py

```diff
@@ -0,0 +1,41 @@
+from .train_options import TrainOptions
+from util.util import MAX_INT
+
+
+class PredictOptions(TrainOptions):
```

Comment:
I suppose it doesn't work if PredictOptions only inherits BaseOptions, this will be fixed in future PR